README.rst

```diff
@@ -1090,9 +1090,23 @@ Example:
                'udp': {'bitrate': '20M'},
                'tcp': {'bitrate': '0'},
            },
-       }
+       },
+       # another org
+       'b9734710-db30-46b0-a2fc-01f01046fe4f': {
+           # available iperf servers
+           'host': ['iperf.openwisp2.io', '192.168.5.3'],
+           'client_options': {
+               'port': 5207,
+               'udp': {'bitrate': '50M'},
+               'tcp': {'bitrate': '20M'},
+           },
+       },
    }
 
+**Note:** If an organisation has more than one iperf server configured, then it enables
```

Comment:
organisation > organization, the spelling with the s is valid too but let's stay consistent.